server/api/auth.py

```diff
@@ -0,0 +1,150 @@
+import functools
+import json
+from sqlalchemy.orm.exc import NoResultFound
+from flask import Blueprint as bp
+from flask import (
+    make_response, redirect,
+    request, session,
+    url_for, g
```

Comment:
looks like you're not using redirect and url_for

Reply:
Alright, I'll change that.